packages/snaps-controllers/src/interface/SnapInterfaceController.test.ts

```diff
@@ -0,0 +1,228 @@
+import { form, input } from '@metamask/snaps-sdk';
+import { MOCK_SNAP_ID } from '@metamask/snaps-utils/test-utils';
+
+import {
+  getRestrictedSnapInterfaceControllerMessenger,
+  getRootSnapInterfaceControllerMessenger,
+} from '../test-utils';
+import { SnapInterfaceController } from './SnapInterfaceController';
+
+describe('SnapInterfaceController', () => {
+  describe('createInterface', () => {
+    it('can create a new interface', () => {
+      const rootMessenger = getRootSnapInterfaceControllerMessenger();
+      const controllerMessenger =
```

Comment:
Please test the actions as well or simply replace the direct function calls with the actions